src/main/java/net/greatstart/dao/hibernate/ProjectDaoImpl.java

```diff
@@ -2,34 +2,61 @@
 
 import net.greatstart.dao.ProjectDao;
 import net.greatstart.model.Project;
+import org.hibernate.Session;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.hibernate.Session'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)